Source/WebKit/UIProcess/Extensions/WebExtension.h

```diff
@@ -238,6 +238,7 @@ class WebExtension : public API::ObjectImpl<API::Object::Type::WebExtension>, pu
     bool hasPageAction();
 
 #if ENABLE(WK_WEB_EXTENSIONS_SIDEBAR)
+    bool hasSidebar();
```

Comment:
You should consider exposing this on `WKWebExtension` too, with an underscore in `WKWebExtensionPrivate.h` for now.